keyboards/mode/eighty/config.h

```diff
@@ -0,0 +1,106 @@
+/*
+Copyright 2018 Alexander Fougner <fougner89 at gmail.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x00DE
```

Comment:
This isn't used, but the low value makes me uneasy.

Reply:
What problem could there be? I didn't choose it, but I see no issues there

Reply:
I changed it to 0x6869 in 60629e5